File: PresentationWeb/src/models/PilotRaceStatsModel.ts
export class PilotRaceStatsModel {
  public pilotId: string;

  public name: string;

  public position: number;

  public meanVelocity: number;

  public totalRaceTime: string; // only display there's no operation

  public lapRaceQuantity: number;

  public bestLapRace: number;

  public timeAfterWinner: string; // only display there's no operation
}

File: Application/PilotRaceStatsModel.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace RaceAnalysis.Application
{
    public class PilotRaceStatsModel
    {
        public string PilotId { get; set; }

        public string Name {get; set;}

        public int Position { get; set; }        

        public double MeanVelocity { get; set; }

        public TimeSpan TotalRaceTime { get; set; }

        public int BestLapRace { get; set; }

        public TimeSpan TimeAfterWinner { get; set; }
    }
}
